src/cases/containers/case-list/case-list.component.ts

```diff
@@ -60,7 +60,7 @@ export class CaseListComponent implements OnInit, OnDestroy {
   constructor(
     public store: Store<fromCaseList.State>,
     private appConfig: AppConfig,
-    private definitionsService: DefinitionsService,
+    private definitionsService: DefinitionsService
```

Comment:
You might be able to leave the trailing comma in here, up to you :) I like the use of trailing comma's as it means there are less merging issues. https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8